roles/dellhw_exporter/vars/main.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+_dellhw_exporter_helm_values:
+  image:
+    repository: "{{ dellhw_exporter_image | atmosphere.common.docker_image('name') }}"
+    tag: "{{ dellhw_exporter_image | atmosphere.common.docker_image('tag') }}"
+  psp:
+    create: false
```

Comment:
I think this is not necessary because based on the author of the chart they said they made a new release which fixed this issue.

Reply:
will get this error if set to true
```
 Helm install failed for release monitoring/prometheus-dellhw-exporter with chart prometheus-dellhw-exporter@1.0.1: unable to build kubernetes objects from release manifest: resource mapping not found for name: "prometheus-dellhw-exporter" namespace: "" from "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1"